wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -301,13 +321,61 @@ default ProxyCommand asProxy() {
    * running and the condition changes to true, the command will not stop running. The requirements
    * of this command will be kept for the new conditional command.
    *
+   * <p>Note: This decorator works by adding this command to a composition. The command the
+   * decorator was called on cannot be scheduled independently or be added to a different
+   * composition (namely, decorators), unless it is manually cleared from the list of composed
+   * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
+   * returned from this method can be further decorated without issue.
+   *
    * @param condition the condition that will prevent the command from running
    * @return the decorated command
+   * @see #skipIf(BooleanSupplier)
+   * @see #onlyIf(BooleanSupplier)
    */
   default ConditionalCommand unless(BooleanSupplier condition) {
     return new ConditionalCommand(new InstantCommand(), this, condition);
   }
 
+  /**
+   * Decorates this command to only run if this condition is not met. If the command is already
+   * running and the condition changes to true, the command will not stop running. The requirements
+   * of this command will be kept for the new conditional command.
+   *
+   * <p>Note: This decorator works by adding this command to a composition. The command the
+   * decorator was called on cannot be scheduled independently or be added to a different
+   * composition (namely, decorators), unless it is manually cleared from the list of composed
+   * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
+   * returned from this method can be further decorated without issue.
+   *
+   * @param condition the condition that will prevent the command from running
+   * @return the decorated command
+   * @see #unless(BooleanSupplier)
+   * @see #onlyIf(BooleanSupplier)
+   */
+  default ConditionalCommand skipIf(BooleanSupplier condition) {
```

Comment:
How is this different from `unless`?

Reply:
Why do we need an alias here?

Previous experience with aliases shows that it causes confusion and people looking for differences that aren't there (that's what happened with Trigger/Button).

Reply:
From a conversation in programming-discussion with @SamCarlberg it could be a more readable alternative in sequential command groups (`sequence(commandA().skipIf(...), commandB())` instead of `sequence(commandA().unless(...), commandB())`), and hopefully documenting that it's just an alias would help (though I don't know if that was already tried with Trigger/Button).

Reply:
That doesn't seem more readable, ngl.

What happened with Trigger/Button was that there were two names for everything, and people expected there to be a difference where there was none. It proves Zen of Python's "there should only be one way of doing something".

Reply:
`skipIf` was more a suggestion for command compositions to read a bit easier

```java
commandA()
  .andThen(commandB())
  .andThen(commandC().skipIf(() -> something))
```

I'm ambivalent about its actual use, and I agree that it could cause confusion to have multiple ways to do the same thing.

Reply:
I'll remove it for now, `unless` is usable enough anyways.